src/components/molecules/Assignment/Assignment.tsx

```diff
@@ -79,6 +79,8 @@ const Assignment: FC<AssignmentProps> = ({
     [vendor]
   ) as DiscountPercentages
 
+  const feature = job_definition.job_key
```

Comment:
If there isn't any job_key,  this part might give as an error

Reply:
Dima confirmed that it can never be null